graphql-server/src/databases/database.resolver.ts

```
@@ -76,6 +82,26 @@ export class DatabaseResolver {
     return conn.currentDatabase();
   }
 
+  @Query(_returns => DatabaseConnection, { nullable: true })
+  async currentConnection(): Promise<DatabaseConnection | undefined> {
+    const config = this.conn.getWorkbenchConfig();
+    if (!config) return undefined;
+    const isDolt = this.conn.getIsDolt();
+    const storedConnections = await this.storedConnections();
+    const connectionName =
+      storedConnections.find(x => x.connectionUrl === config.connectionUrl)
+        ?.name || "connections";
```

Seems like we want to return an error or undefined if this doesn't match?